### PR TITLE
Update README.rst

### PR DESCRIPTION
**Pull request by glensc**
add information about excluding files.

sample taken from
https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/1767#issuecomment-206833220

### CODE REVIEW

**Comment by keradus** [comment]
Please read two last points of https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/1.11/CONTRIBUTING.md#quick-guide

**Comment by glensc** [comment]
> Regenerate readme: php php-cs-fixer readme > README.rst. Do not modify README.rst manually!

blah. annoying. but the changeset itself is okay?

**Comment by keradus** [comment]
not annoying, as we are autogenerating a lot of file's content ;)

File: README.rst [hunk 1]
```diff
@@ -692,6 +693,8 @@ to the default list of symfony-level fixers:
         ->finder($finder)
     ;
 
+NOTE: `exclude` will work only for directories, so if you need to exclude file, try `notPath`.
```
Comment (keradus) [inline README.rst]: I would just link to symfony/finder docs.
For now it says about some functionallity, but not all. And recreating Symfony docs here is pointless.

**Comment by glensc** [comment]
you omitted answer to question, do i assume `yes` then?

**Comment by keradus** [comment]
My answer is around diff line it concerns

File: README.rst (hunk 1 above)
Comment (glensc) [inline README.rst]: symfony docs are hard to read, they are written probably in api point of view not actual real life examples, like here. and it also does not emphasis that `exclude` **does not work** for files.

Comment (glensc) [inline README.rst]: maybe add `.gitignore` style matching? it is more stright forward to write and understand:

```
src/**.php
!src/Subdir
*.php
```

**Comment by glensc** [comment]
updated commit.

File: README.rst (hunk 1 above)
Comment (keradus) [inline README.rst]: No. We do not have (and do not want to have) our own finder.
For us finder is anything that will implement `\Traversable`.

symfony/finder is only example

**Comment by SpacePossum** [comment]
:+1:

**Comment by keradus** [comment]
Thank you @glensc

**Pull request closed without merging**